packages/core/src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -219,8 +124,15 @@ export class DropdownMenu {
 
     if (this.isToggleButtonEvent(composedPath)) {
       this.open = !this.open;
+      this.popoverElement?.togglePopover();
```

Comment:
Je zou kunnen stellen dat er op deze manier twee "stores" zijn waar de "open" state wordt bijgehouden: Namelijk lokaal op `this.open`, en in het popover element.

Wat ik voorstel is de ene laten luisteren naar de ander: `togglePopover()` accepteert een `force` parameter. Als we daar `this.open` meegeven lopen ze nooit uit de pas:

```suggestion
      this.popoverElement?.togglePopover(this.open);
```

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/togglePopover#force

Een ander fraai patroon zou een setter/instance method zijn. Er zijn een aantal plekken waar `this.open` wordt gezet, en ALTIJD gevolgd door `hidePopover()`.

```ts
toggleOptions(open?: boolean) {
  this.open = !open;
  this.popoverElement?.togglePopover(this.open);
```

Ik heb niet scherp hoe/of Stencil getter/setters op `@State()` properties ondersteund, gemakshalve heb ik een voorbeeld met instance method gemaakt.

Reply:
Ik heb dit gedaan en zelfs 

```tsx
if (!this.open && this.cleanUp) {
   this.cleanUp();
   this.cleanUp = undefined;
}
```
toegevoegd aan de nieuwe instance-method `toggleOptions`